### DISCUSSION TITLE
Add Search To Organizations Repos

### DISCUSSION THREAD

**Discussion by paulcruse-syn** [community/community#34722] [Mobile]
<details>
<summary>Search Is almost useless in current form. Should be able to just search repos in my organization, just like the web, not the entire GitHub community. </summary>
GitHub for iOS, Version 1.82.0 (79366424)
Device: iPhone 13 mini (iOS 16.0)
</details>

**Comment by jfullstackdev** [comment]
there is that feature

![image](https://user-images.githubusercontent.com/47092464/193394415-765c8c6b-a2e7-43fb-be54-e3175a6d04d1.png)

when you go to the Org page,
